src/main/java/touroll/speech/SpeechElement.java

```diff
@@ -0,0 +1,29 @@
+package touroll.speech;
+
+import touroll.VideoElement;
+
+import java.io.File;
+import java.io.IOException;
+
+public class SpeechElement implements VideoElement
+{
+    private SpeechRenderer renderer;
+    private File source;
+    private File output;
+
+    public SpeechElement(SpeechRenderer renderer, File source, File output) {
```

Comment:
`output` isn't passed in the constructor.
` output` is generated from `source` in the prepare() method